packages/components/src/templates/teaser-with-link.stories.ts

```diff
@@ -1,189 +0,0 @@
-import '../solid-components';
```

Comment:
the original link on the component itself needs to be updated or removed. It is still linking to the "teaser with link" template.